wrapanapi/systems/msazure.py

```diff
@@ -1130,6 +1134,21 @@ def delete_stack_by_date(self, days_old, resource_group=None):
                              result)
         return results
 
+    def delete_compute_image_by_name(self, resource_group=None, image_list=None):
+        """
+        Delete compute images by name
+
+        :resource_group: (str) Name of the resource group. "FooBar"
+        :image_list: (list) List of images. ["imagename1", "imagename2", "imagename3"]
+        """
+        result = []
+        resource_group = resource_group or self.resource_group
+        for image in image_list:
+            self.logger.info("Deleting {} from {}".format(image, resource_group))
```

Comment:
If I'm being picky, proper use here is `self.logger.info("Deleting '%s' from '%s'", image, resource_group)` so that the logger formatting takes effect properly.